hardware/raspberrypi/bootmodes/net_tutorial.md

```diff
@@ -35,6 +35,18 @@ Ensure the output `0x3020000a` is correct.
 
 The client configuration is almost done. The final thing to do is to remove the `program_usb_boot_mode` line from `config.txt` (make sure there is no blank line at the end). You can do this with `sudo nano /boot/config.txt`, for example. Finally, shut the client Raspberry Pi down with `sudo poweroff`.
 
+## Client configuration (Raspberry Pi 3B+ / 4)
+
+These instructions only apply to the Raspberry Pi 3B+ and 4.
```

Comment:
The Raspberry Pi 3B+ doesn't have a bootloader eeprom...

Reply:
Oh you're quite right, how silly of me!  I will see if I can find out what is required on the 3B+ and update this PR.

Reply:
IIRC the 3B+ supports network (and USB) booting out-of-the-box?

Reply:
> IIRC the 3B+ supports network (and USB) booting out-of-the-box?

Correct. There's no configuration needed - if it hasn't found an SD card in 5 seconds it moves to USB host boot and tries USB MSD and net, as already stated on this page.